src/test/regress/expected/local_shard_execution.out

```diff
@@ -346,15 +352,17 @@ EXPLAIN (ANALYZE, COSTS OFF, SUMMARY OFF, TIMING OFF) DELETE FROM distributed_ta
 ---------------------------------------------------------------------
  Custom Scan (Citus Adaptive) (actual rows=0 loops=1)
    Task Count: 1
+   Data received from workers: 0 bytes
    Tasks Shown: All
    ->  Task
+         Data received from worker: 0 bytes
```

Comment:
this seems a little redundant

Reply:
I changed the code to only show this explain section when we expect results from the query, i.e. a tupledescriptor with a nonzero number of attributes.